### ISSUE TITLE
The server does not start

### ISSUE THREAD

**Issue by yannham** [fannheyward/coc-rust-analyzer#297]
The coc-rust-analyzer server is unable to start. I was having problems (like go to definition has not been working for days now), so I tried to uninstall everything including coc and do a fresh reinstall, removing all artifacts from ~/.config/coc. However now the server does not even start.

My coc settings:

```
{
  "languageserver": {
    "haskell": {
      "command": "ghcide",
      "args": [
        "--lsp"
      ],
      "rootPatterns": [
        ".stack.yaml",
        ".hie-bios",
        "BUILD.bazel",
        "cabal.config",
        "package.yaml"
      ],
      "filetypes": [
        "hs",
        "lhs",
        "haskell"
      ]
    },
    "ocaml-lsp": {
      "command": "opam",
      "args": ["config", "exec", "--", "ocamllsp"],
      "filetypes": ["ocaml", "reason"]
    }
  }
}
```

Logs:

```
## versions

vim version: NVIM v0.4.3
node version: v12.18.0
coc.nvim version: 0.0.78
coc.nvim directory: /home/XXX/.vim/bundle/coc.nvim
term: xterm-256color
platform: linux

## Output channel: Rust Analyzer Language Server Trace


## Output channel: rust-analyzer
[Error  - 17:38:17] Connection to server is erroring. Shutting down server.
[Error  - 17:38:17] Connection to server is erroring. Shutting down server.
[Error  - 17:38:17] Connection to server is erroring. Shutting down server.
/home/yago/.config/coc/extensions/coc-rust-analyzer-data/rust-analyzer: 3: Syntax error: "(" unexpected
[Error  - 17:38:17] /home/yago/.config/coc/extensions/coc-rust-analyzer-data/rust-analyzer exited with code: 2

```

Rustc/cargo version: 1.44.1

**Comment by jubos** [comment]
I encountered this as well and fixed it in the short term by downloading a rust-analyzer release for your OS from https://github.com/rust-analyzer/rust-analyzer/releases/tag/2020-06-22 and replacing the rust-analyzer binary.

**Comment by yannham** [comment]
Thanks a lot, this allows me to at least work normally in the meantime. For the maintainer information, I tried to downgrade coc-rust-analyzer to 7.6 without success.

**Comment by GopherJ** [comment]
![image](https://user-images.githubusercontent.com/33961674/85626073-bfdee300-b66c-11ea-9971-ea3cc4528224.png)

**Comment by GopherJ** [comment]
downloading a binary works but it's slowing down my vim. any one the same issue?

**Comment by GopherJ** [comment]
It's interesting that when I hit **K** on close `}` symbol. Let's say `async fn main`, neovim freezes abd I got the following error:

![image](https://user-images.githubusercontent.com/33961674/85630675-5a8ef000-b674-11ea-947b-5c6090e78325.png)

**Comment by GopherJ** [comment]
@fannheyward it seems `K` can crash `coc-rust-analyzer`, any idea on this? I've basically tested on completion and documentation on a personal project.

Completion isn't stable, sometimes it works great sometimes there isn't any completion.
Documentation currently crashes. I have no idea yet

**Comment by fannheyward** [comment]
@yannham does `/home/yago/.config/coc/extensions/coc-rust-analyzer-data/rust-analyzer --version` works?

**Comment by fannheyward** [comment]
@GopherJ can't reproduce the `K` error, properly is LS error.

**Comment by yannham** [comment]
@fannheyward it seems it does not download the binary for the right platform as it cannot even execute, I get a "exec() format error" (Bash return code 126). I'm on Linux 5.6.0-2-amd64 #1 SMP Debian 5.6.14-2 (2020-06-09) x86_64 GNU/Linux.

**Comment by fannheyward** [comment]
@yannham looks like the binary downloaded is broken. I've just released coc-rust-analyzer v0.7.8, please give it a try.

**Comment by yannham** [comment]
@fannheyward fixed by 0.7.8 !
